### PR TITLE
Fixes a regression in the AWS intialization script for Dev and Test venue

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary

The option --terrgrunt was not supported anymore.

### 🤖 AI Assistance Disclosure
- [ ] No AI assistance used
- [ ] AI used for light assistance (e.g., suggestions, refactoring, documentation help, minor edits)
- [ ] AI used for moderate content generation (AI generated some code or logic, but the developer authored or heavily revised the majority)
- [x] AI generated substantial portions of this code

Estimated % of code influenced by AI: __100_ %

## ⚙️ Test Data and/or Report

Tested the full deployment with integration test in the dev venue.

## ♻️ Related Issues

Necessary for the deployment in test https://github.com/NASA-PDS/registry/issues/542

## 🤓 Reviewer Checklist

*Reviewers: Please verify the following before approving this pull request.*

### Documentation and PR Content
- [ ] **Documentation:** README, Wiki, or inline documentation (Sphinx, Javadoc, Docstrings) have been updated to reflect these changes.
- [ ] **Issue Traceability:** The PR is linked to a valid GitHub Issue
- [ ] **PR Title:** The PR title is "user-friendly" clearly identifying what is being fixed or the new feature being added, that if you saw it in the Release Notes for a tool, you would be able to get the gist of what was done.

### Security & Quality
- [ ] **SonarCloud:** Confirmed no new High or Critical security findings.
- [ ] **Secrets Detection:** Verified that the Secrets Detection scan passed and no sensitive information (keys, tokens, PII) is exposed.
- [ ] **Code Quality:** Code follows organization style guidelines and best practices for the specific language (e.g., PEP 8, Google Java Style).

### Testing & Validation
- [ ] **Test Accuracy:** Verified that test data is accurate, representative of real-world PDS4 scenarios, and sufficient for the logic being tested.
- [ ] **Coverage:** Automated tests cover new logic and edge cases.
- [ ] **Local Verification:** (If applicable) Successfully built and ran the changes in a local or staging environment.

### Maintenance
- [ ] **Backward Compatibility:** Confirmed that these changes do not break existing downstream dependencies or API contracts (or that breaking changes are clearly documented).

### CODE REVIEW

File: terraform/run-init-on-aws.py [hunk 1]
```diff
@@ -61,9 +61,10 @@ def print_section(self, emoji: str, message: str):
 
     def check_terraform_directory(self) -> bool:
         """Check that the opensearch_serverless and applications Terraform state directories exist."""
+        marker = "terragrunt.hcl" if self.tf_cmd == "terragrunt" else "main.tf"
         for tf_dir in (self.tf_opensearch_dir, self.tf_applications_dir):
-            if not (tf_dir / "main.tf").exists():
-                print(f"Error: Terraform directory not found or missing main.tf: {tf_dir}")
+            if not (tf_dir / marker).exists():
+                print(f"Error: Terraform directory not found or missing {marker}: {tf_dir}")
```
Comment (nutjob4life) [inline terraform/run-init-on-aws.py:67]: In general diagnostic messages should go to the standard error:
```python
print(
    f"Error: Terraform directory not found or missing {marker}: {tf_dir}",
    file=sys.stderr
)
```

**Review by nutjob4life — APPROVED** [review]
Code delta: ✓
One concern: not using `sys.stderr`
Is it minor? ✓
Approval: ✅
Caveat: review based on visual inspection only;  trust accordingly
